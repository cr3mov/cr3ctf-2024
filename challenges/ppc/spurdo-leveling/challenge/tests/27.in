179412610517856625 909432408208685590
537956 621697383959595
537956 40871614030104869
