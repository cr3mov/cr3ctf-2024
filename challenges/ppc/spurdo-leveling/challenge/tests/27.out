4106476 18 18
